webui/react/src/hooks/useModal/JupyterLab/useModalJupyterLab.tsx

```diff
@@ -45,7 +54,7 @@ const settingsConfig: SettingsConfig<JupyterLabOptions> = {
       type: union([number, undefinedType]),
     },
     template: {
-      defaultValue: '',
+      defaultValue: undefined,
```

Comment:
@EmilyBonar I believe this resolves the bug with templates you mentioned, thanks for bringing it up!